backend/internal/adapters/secondary/repositories/user.go

```diff
@@ -91,6 +94,31 @@ func (r *userRepository) Add(user *users.User) (int64, error) {
 		return 0, fmt.Errorf("unable to insert user: %w", err)
 	}
 
+	// Newly registered users are automatically friends with Gretel
+	var firstUserID, secondUserID int64
+	if constants.GretelUserId < userId {
+		firstUserID = constants.GretelUserId
+		secondUserID = userId
+	} else {
+		firstUserID = userId
+		secondUserID = constants.GretelUserId
+	}
```

Comment:
Do we need this? Gretel should always come first, right? Else, something went wrong with our assumptions already.

Reply:
hmm, just wondering if there would be the case where we changed the id of gretel (or added another gretel-equivalent)